### PR TITLE
Restricted plan changes

### PR DESCRIPTION
**Pull request by dzucconi**
cancel-reenable_manually

cancel-reenable

### CODE REVIEW

**Comment by dzucconi** [comment]
Depends on the simultaneous deploy of https://github.com/aredotna/api/pull/137

File: apps/settings/components/payment_methods/view.coffee [hunk 1]
```diff
@@ -74,13 +74,13 @@ module.exports = class PaymentMethodsView extends Backbone.View
 
     $target.text 'Saving'
 
-    @model.save(default_source: $target.data('id'))
+    @model.save(default_soturce: $target.data('id'))
```
Comment (dzucconi) [inline apps/settings/components/payment_methods/view.coffee]: typo...

**Comment by dzucconi** [comment]
PSA: Next time I have to touch this it gets a full test suite before I write a single new line of code.

File: apps/settings/components/payment_methods/view.coffee [hunk 2]
```diff
@@ -101,7 +101,7 @@ module.exports = class PaymentMethodsView extends Backbone.View
 
         $target.text 'Removed!'
 
-      .catch (error) =>
+      , (error) =>
```
Comment (broskoski) [inline apps/settings/components/payment_methods/view.coffee]: These are not real promises?

Comment (dzucconi) [inline apps/settings/components/payment_methods/view.coffee]: The whole sync layer is a mess.

File: apps/settings/components/your_plan/index.jade [hunk 3]
```diff
@@ -6,19 +6,18 @@
     .form-card__field
       label Current Plan
       .bordered-select
-          select( name='plan', disabled=(customer.get('is_lifetime') ? 'disabled' : null) )
+          select( name='plan', disabled=(customer.get('is_premium') ? 'disabled' : null) )
```
Comment (broskoski) [inline apps/settings/components/your_plan/index.jade]: 👍

Comment (dzucconi) [inline apps/settings/components/your_plan/index.jade]: How long do you think it'll be until we get an email asking to switch plans

Comment (dzucconi) [inline apps/settings/components/your_plan/index.jade]: I'm guessing 1 day.

Comment (broskoski) [inline apps/settings/components/your_plan/index.jade]: :|

**Comment by broskoski** [comment]
Nice work, merge when ready!

**Pull request merged**